types/queries.d.ts

```diff
@@ -88,6 +88,11 @@ export interface ByRoleOptions extends MatcherOptions {
    * pressed in the accessibility tree, i.e., `aria-pressed="true"`
    */
   pressed?: boolean
+  /**
+   * If true only includes elements in the query set that are marked as
+   * current in the accessibility tree, i.e., `aria-current="true"`
+   */
+  current?: boolean
```

Comment:
I would rather just query against exact values. Then we automatically support newly introduced values. It's not clear to me that you want `aria-current="page"` to match against `current: true` as well as `aria-current="date"`. Would people actually want `aria-current="page"` and `aria-current="date"` in a `getAllByRole(role, { current: true })` query?

Reply:
It was one of the solution suggestions at the original issue. Both suggestions have it pros and cons.

Now we have two options:
1. Merge current implementation and open additional issue to improve/enhance it to yu
2. Fix it right here in this MR. I think it preferred way. I'll try to do it on weekend.

Thanks for the feedback.